packages/core/test/portal/portalTests.tsx

```diff
@@ -30,6 +30,19 @@ describe("<Portal>", () => {
         assert.lengthOf(document.getElementsByClassName(CLASS_TO_TEST), 1);
     });
 
+    it("attaches contents to specified container", () => {
```

Comment:
technically, this and the `Overlay` test are the only ones that really matter, since all the other components just use those two code paths. but having those tests (especially if you slim them down as i suggested) won't hurt.